src/detection/terminalshell/terminalshell_linux.c

```diff
@@ -226,6 +226,13 @@ static void getTerminalFromEnv(FFTerminalResult* result)
         getenv("ALACRITTY_WINDOW_ID") != NULL
     ) term = "Alacritty";
 
+    //Ghostty
+    else if(
+        getenv("GHOSTTY_RESOURCES_DIR") != NULL ||
+        getenv("GHOSTTY_SHELL_INTEGRATION_NO_SUDO") != NULL ||
+        getenv("GHOSTTY_BIN_DIR") != NULL
+    ) term = "Ghostty";
```

Comment:
Ghostty reports $TERM_PROGRAM_VERSION, so additional env-var checks are not needed.